core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpDocServiceTest.java

```diff
@@ -102,6 +107,11 @@ protected void configure(ServerBuilder sb) throws Exception {
 
     @Test
     public void jsonSpecification() {
+        /*try {
+            Thread.sleep(Long.MAX_VALUE);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }*/
```

Comment:
How about adding a static final boolean field like: `boolean DEMO_MODE = false` and sleeping forever when it's `true`?